data-science-pipelines-operator/base/params.env

```diff
@@ -6,7 +6,8 @@ IMAGES_CACHE=registry.access.redhat.com/ubi8/ubi-minimal@sha256:e52fc1de73dc2879
 IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro@sha256:443db9a646aaf9374f95d266ba0c8656a52d70d0ffcc386a782cea28fa32e55d
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103@sha256:cafc7364494fb7206c373a1235fd5da74399c19b5c34d87dd02aa07e8f343fa2
 IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator@sha256:bd4f3cfc9688aeb4296a5f3f7274557adeca0a8811533da750f05b485a819a8d
-IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy@sha256:d0f2f1ef0bdc3aa1a70794ac8ac779271b634af83e939029ac5224ec0c815d7a
+IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy@sha256:ab112105ac37352a2a4916a39d6736f5db6ab4c29bad4467de8d613e80e9bb33
```

Comment:
@shalberd dsp is using 4.12 oauth proxy, so this change is not needed on the dsp front since the primary reason for the issue is to stop using old oauth proxy images afaik

also, typically we make changes in the dsp manifests [within the component repo](https://github.com/opendatahub-io/data-science-pipelines-operator/tree/main/config) before trickling the changes down to odh-manifests (as these need to be kept in sync)

Reply:
@HumairAK no, the idea of this PR is to unify the version of ose-oauth-proxy across all components. I know some people use latest versions of Openshift i.e. 4.12 or 4.13, but the standard here is really OCP 4.10. Thus the unified reference to v4.10 of ose-oauth-proxy.

We use digest format for disconnected support. You are right about also having to do this in your repo as well. ODH Dashboard has already unified to v4.10 from old 4.8 in their repo, for example. In a second step, they will now include the image in digest format.

Reply:
no hard requirement, we can go with 4.10, but may we get this merged in the component repo first (at least for dsp)? we can keep this in the pr here in the meantime, would you be willing to make the same adjustment [here](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/base/params.env#L9)?